pkg/metrics/wal/series.go

```diff
@@ -8,46 +8,36 @@ import (
 	"github.com/prometheus/prometheus/tsdb/chunks"
 )
```

Comment:
Should we add a disclaimer similar to what we did when porting promtail over? Basically its ported over with only changes to metric names